azurerm/internal/services/cosmos/cosmosdb_gremlin_graph_resource.go

```diff
@@ -92,6 +92,7 @@ func resourceCosmosDbGremlinGraph() *schema.Resource {
 				Optional:     true,
 				ForceNew:     true,
 				ValidateFunc: validation.StringIsNotEmpty,
+				Default:      "/_partitionKey",
```

Comment:
talked this over internal and instead of a default lets just leave this as required so the user can decide what to do rather then have magic upgrade behaviour